sample-azure-ha-yamls/azure-ha-docker-service.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: nxrm-nexus-docker-service
+  namespace: nxrm
+  labels:
+    app: nxrm
+spec:
+  type: NodePort
+  selector:
+    app: nxrm
+  ports:
+    - name: docker-service
+      protocol: TCP
+      port: 9090
```

Comment:
Whats port 9090 for?

Reply:
It's the port to forward traffic to if you set up a separate ingress for docker.  We've recommended using docker sub domain or reverse proxy if you have more than one docker repository